cmd/juju/application/deploy_test.go

```diff
@@ -1021,6 +1018,34 @@ func (s *charmstoreSuite) assertDeployedApplicationBindings(c *gc.C, info map[st
 	}
 }
 
+// assertRelationsEstablished checks that the given relations have been set.
+func (s *charmstoreSuite) assertRelationsEstablished(c *gc.C, relations ...string) {
```

Comment:
Just check we need both of these new functions - we need one now, maybe we will need both eventually?

Reply:
These were copied verbatim from `legacyCharmStoreSuite`. They're there to maintain current test behaviour.